tb_db/models.py

```diff
@@ -134,18 +136,32 @@ class MiruProfile(Base, HistoryMixin):
     miru_pattern = Column(String)
 
 
-class CgmlstCluster(Base, HistoryMixin):
+class CgmlstCluster(Base):
 
     cluster_id = Column(String)
 
-    samples = relationship(
-        "Sample", back_populates="cgmlst_cluster", cascade="all, delete-orphan"
-    )
+    #samples_cgmlst_cluster = relationship(
+    #    "SampleCgmlstCluster", back_populates="cgmlst_cluster", cascade="all, delete-orphan"
+    #)
 
 
-class MiruCluster(Base, HistoryMixin):
+class MiruCluster(Base):
 
     cluster_id = Column(String)
-    samples = relationship(
-        "Sample", back_populates="miru_cluster", cascade="all, delete-orphan"
-    )
+    #samples_miru_cluster = relationship(
+    #    "SampleMiruCluster", back_populates="miru_cluster", cascade="all, delete-orphan"
+    #)
+
+
+class SampleCgmlstCluster(Base):
```

Comment:
I'm not certain about this, but I'm not sure that it's necessary to explicitly create a `SampleCgmlstCluster` or `SampleMiruCluster` model class.

Consider using the Many-to-Many pattern described here:

https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#many-to-many